explainer.md

```diff
@@ -488,6 +586,48 @@ without jank.
 </div>
 &nbsp;
 
+---
+### Active vs passive locks
+
+In order to allow the use of display locking to animate content, we need to
+consider the timing of the updates. The current timing landscape looks like the
+following:
+
+* Frame 1: acquireDisplayLock is called, which needs to finish current update
+  phases and stash the paint.
+* Frame 2: the given callback is called and the intent of the script is to
+  present the contents.
+* Frame 2 also detects that no more things have been scheduled and processes the
+  update phases to completion.
+* End of Frame 2: the promise is resolved. Note that here, script cannot
+  re-acquire the lock, since that would prevent the visual updates from making
+  it to screen; it must do this in a rAf or a setTimeout callback instead.
+* Frame 3: script calls acquireDisplayLock again, which has to finish the
+  current update phases and stash the paint output (as in the first step).
+* Frame 4: the callback is run, etc.
+
+If this animation continues like this, there is always two frames between script
+successively changing the animated properties (Frames 2 and 4 are the ones that
+run the callback). This means that the best frame rate the script may achieve
+with this version of display locking is 30fps.
+
+Since this is a common use case, we need to allow for a display lock driven
+animation to run at 60fps instead.
+
```

Comment:
Also add a paragraph about "async subtree" (mode 1) vs "async element + async subtree" (mode 2)

Reply:
Note sure what you mean here? In what context do I bring up this difference? Or do you mean like a separate section?

Reply:
In a separate example/paragraph..